src/app/nodetool/qgsmaptoolnodetool.cpp

```diff
@@ -114,6 +114,11 @@ void QgsMapToolNodeTool::canvasMoveEvent( QgsMapMouseEvent* e )
   {
     if ( mMoveRubberBands.empty() )
     {
+      if ( mSelectRubberBand )
```

Comment:
needless to check, just delete and set to 0. If it's alread 0 nothing will happen.